icu_benchmarks/models/wrappers.py

```
@@ -3,6 +3,7 @@
 from typing import Dict, Any, List, Optional, Union
 from pathlib import Path
 import torchmetrics
+from ignite.metrics import EpochMetric
```

Ruff (0.8.2)

`ignite.metrics.EpochMetric` imported but unused

Remove unused import: `ignite.metrics.EpochMetric`

(F401)